sidecar/internal/util/util_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2021 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+)
+
+func TestValidateControllerEndpoint(t *testing.T) {
+	type args struct {
+		endpoint string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "valid case 1",
+			args: args{
+				endpoint: "192.168.61.228:8080",
+			},
+			wantErr: false,
+		},
+		{
+			name: "valid case 2",
+			args: args{
+				endpoint: "172.0.61.22:8000",
```

Comment:
What is the difference with the 1st case?

Do not use 172.0.x.x as example, use an address from the ranges listed at https://datatracker.ietf.org/doc/html/rfc1918#section-3